content/techniques/configuration.md

```diff
@@ -535,3 +535,31 @@ You can then use it as usual, by calling the `get` method with the configuration
 ```typescript
 const port = configService.get('PORT');
 ```
+
+### Using in e2e/integration tests
+
+It is also possible to utilize the Configuration Service in testing. In the following example we import it into our module and then retrieve the static instance, assigning it to a `configService` variable that can be used in our tests.
+
+```typescript
+@@filename(cats.controller.spec)
+import { Test } from '@nestjs/testing';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { CatsController } from './cats.controller';
+import { CatsService } from './cats.service';
+
+describe('CatsController', () => {
+    let controller: CatsController;
+    let configService: ConfigService;
+
+    beforeEach(async () => {
+        const moduleRef = await Test.createTestingModule({
+            imports: [ConfigModule.forRoot()],
+            controllers: [CatsController],
+            providers: [CatsService],
+        }).compile();
+
+        controller = moduleRef.get<CatsController>(CatsController);
+        configService = moduleRef.get<ConfigService>(ConfigService);
+    });
+});
+```
```

Comment:
I'm looking at this example setup and honestly, I'm not sure if this really brings any extra value, given that there's nothing specific regarding `ConfigModule` and the way you use `ConfigService` in e2e/integration/any tests. Basically, the fact that you must import a module to make it available in tests should be obvious after reading this chapter https://docs.nestjs.com/fundamentals/testing (perhaps we need some further tweaks & improvements here?). Likewise, it should be pretty clear from this section https://docs.nestjs.com/fundamentals/testing#end-to-end-testing that `get()` is being used to retrieve **any** statically-scoped provider registered in app/test module fixture. I might be missing something(?) but I would love to know what we could potentially improve to make it easier, especially for newcomers cc @jmcdo29

Reply:
I'm with you on this on Kamil. But it's also difficult for me to know what is and isn't inherently clear to others anymore after working with Nest for so long. It all seems like second nature to me. 

@cjmyles as the author of the PR, would it be correct to assume that this wasn't immediately clear, with reading the above pages?